examples/tpm2-integration-test/integration_test.go

```diff
@@ -0,0 +1,57 @@
+package tpm2_integration_test
```

Comment:
I don't think we need tests for example code.
Especially something like this that requires a lot of manual setup

Reply:
I'd like an easy way to ensure that I haven't broken an example flow...without tests, I don't have this assurance. Also these flows will get a bit more complicated when I submit the code for sealing data. The extra manual setup is installing the example binaries and pointing the test at their directory...this is fairly lightweight.